app/src/main/java/com/example/NotificationCenter.java

```diff
@@ -0,0 +1,39 @@
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class NotificationCenter {
+
+    // Method to show the custom toast notification
+    public static void showLotteryToast(Context context, String status) {
+        
+        LayoutInflater inflater = LayoutInflater.from(context);
+        View layout = inflater.inflate(R.layout.notification_center, null);
+
+        // Get references to the TextViews and ImageView in the layout
+        TextView toastTitle = layout.findViewById(R.id.notificationTitle);
```

Comment:
Im confused about the naming here. In this class it doesn't seems like you are making a toast (which is fine) but I think that these names should properly represent that this is an textview and not a toast